### ISSUE TITLE
Main window for ra25 is misbehaving

### ISSUE THREAD

**Issue by StephenJRead** [emer/leabra#6]
I just updated to the newest version of gi and then updated leabra.  When I started ra25 the main window opened up at several multiple widths of the size of my main external monitor.  When I clicked the green button in the upper left, it then went full screen mode and fit in the external monitor.  But when I then clicked the green button again, the window went totally black and expanded to many times larger than my external monitor.

**Comment by rcoreilly** [comment]
yeah sorry I haven't had a chance to test the new glfw backend sufficiently yet -- will try to later tonight.

**Comment by rcoreilly** [comment]
btw is this on a mac?  can you run something you've never run before, e.g., one of the textbook sims or something (it should show up at a reasonable size) and then go to GoGi Preferences/ Screen Info and paste in what it says there?

**Comment by StephenJRead** [comment]
Sorry. Yes it is on a Mac. Meant to mention that.
I will try that later today.

Sent from my iPhone

On Nov 27, 2019, at 12:17 AM, rcoreilly <notifications@github.com> wrote:

﻿

btw is this on a mac? can you run something you've never run before, e.g., one of the textbook sims or something (it should show up at a reasonable size) and then go to GoGi Preferences/ Screen Info and paste in what it says there?

**Comment by StephenJRead** [comment]
One thing I did notice is that ra25 is opening on my external monitor, while the version of bg I just opened is opening on my laptop window.  

Here are the Screeninfo results for running bg. 

Screen number: 0 name: Color LCD
<br>    geom: (0,0)-(1440,900), depth: 24, logical DPI: 228, physical DPI: 256.67368, logical DPI scale: 0.8882874, physical size: (285,179)
<br>    device pixel ratio: 2, refresh rate: 60
<br>    orientation: OrientationUnknown, native orientation: OrientationUnknown, primary orientation: OrientationUnknown
<br><br>
Screen number: 1 name: LG Ultra HD
<br>    geom: (0,-1080)-(1920,0), depth: 24, logical DPI: 72, physical DPI: 81.55184, logical DPI scale: 0.882874, physical size: (598,340)
<br>    device pixel ratio: 1, refresh rate: 60
<br>    orientation: OrientationUnknown, native orientation: OrientationUnknown, primary orientation: OrientationUnknown


And here are the screen info results for running ra25

Screen number: 0 name: Color LCD
<br>    geom: (0,0)-(2880,1800), depth: 24, logical DPI: 228, physical DPI: 256.67368, logical DPI scale: 0.8882874, physical size: (285,179)
<br>    device pixel ratio: 2, refresh rate: 60
<br>    orientation: OrientationUnknown, native orientation: OrientationUnknown, primary orientation: OrientationUnknown
<br><br>
Screen number: 1 name: LG Ultra HD
<br>    geom: (0,-1080)-(3840,1080), depth: 24, logical DPI: 144, physical DPI: 163.10368, logical DPI scale: 0.882874, physical size: (598,340)
<br>    device pixel ratio: 2, refresh rate: 60
<br>    orientation: OrientationUnknown, native orientation: OrientationUnknown, primary orientation: OrientationUnknown


Interestingly, the bg version window behaves itself when I click on the green button to change window size, whereas the ra25 version will get extra wide and turn black.
